src/app/upload/upload.component.ts

```diff
@@ -14,28 +14,33 @@ import { AudioVersionModel } from '../shared';
       <div class="uploads">
         <template ngFor let-ft [ngForOf]="version.filesAndTemplates">
           <publish-templated-upload *ngIf="ft.tpl" [template]="ft.tpl"
-            [file]="ft.file" [version]="version"></publish-templated-upload>
+            [file]="ft.file" [version]="version" publishClick></publish-templated-upload>
           <publish-illegal-upload *ngIf="!ft.tpl" [file]="ft.file"
             [version]="version"></publish-illegal-upload>
         </template>
       </div>
     </section>
 
-    <section *ngIf="!version.hasFileTemplates">
-      <div class="uploads" [publishFreeReorder]="version">
-        <div *ngIf="version.noAudioFiles" class="empty">
-          <h4 (click)="upinput.click()">Upload a file to get started</h4>
+    <section publishClick>
```

Comment:
This might just be an issue of not understanding the requirements for #109, but this publishClick directive only works for me when I click the footer area (the area surrounding the upload button.) (Chrome v55)